### ISSUE TITLE
PMID:30089908

### ISSUE THREAD

**Issue by Antonialock** [pombase/curation#2209]
In  PMID:30089908 mat3-Mi is annotated....did you mean to annotate mat1-Mi?

**Comment by ValWood** [comment]
I don't know, I'm still *so* confused by these.

The abstract says
 cytosolic M-cell-specific peptide Mi ????

what do we use for this?

**Comment by Antonialock** [comment]
For M-cell specific peptide Mi you want to annotate to SPBC23G7.17c (mat1-mi)

 the reference strain is in the "m configuration" =has the Mi and Mc genes at mat1 locus

genes at the mat1 locus are expressed (mat1-Mi, mat1-mc)
genes at the mat3 are silent (mat3-Mi, mat3-Mc) - these should have NO GO annotation

Similarly, in an ideal world we would annotate
mat1-Pi and mat1-Pc
for P cell information

however, this is not an ideal world, and we only have one reference strain with Mi and Mc at mat1, and in which mat2 has been deleted. 

For this reason we need to:
1. annotate the silent genes mat2-Pc and mat2-Pi (we have no mat1-Pc and mat1-Pi to annotate to)
2. have an additional mating type contig for mat2 and mat3 - this has the unfortunate consequence that we have 2 copies of mat3-Mi and 2 copies of mat3-Mc
but this doesn't cause too much trouble as we shouldn't annotate mat3 to anything


so to summarize:

mat1 locus - contains the two mating genes that are expressed (mat1-Pc and mat1-Pi in P cells and Mat1-Mi and Mat1-Mc in M cells
mat2 locus - contains the silenced 'P DNA bank' genes mat2-Pc and mat2-Pi (deleted in the reference strain, reconstituted on the mating type contig)
mat3 locus - contains the silenced 'M DNA bank' genes mat2-Mc and mat2-Mi (present in the reference strain, reconstituted on the mating type contig)

**Comment by ValWood** [comment]
I'm looking  at 
https://curation.pombase.org/pombe/curs/46c89f342f117ca5/ro/
to fix but I don't see mat3-Mi  in here?

**Comment by ValWood** [comment]
Can we put thin in the documentation somewehre? or as a FAQ?

**Comment by Antonialock** [comment]
Sorry for the confusion, mat3-Mm is a synonym to mat3-Mi

I'll go and rename it now.

**Comment by Antonialock** [comment]
what do you want in the faq? the 'why mat2-pc and mat2-pi are annotated' and 'why there are two copies of mat3-mc and mat3-mi'? (anything else?) I can draft

**Comment by ValWood** [comment]
I don't know the question, but this sounds along the right lines ;)

Do I still need to change anything in the session?

**Comment by Antonialock** [comment]
Not anymore, I did the changes :) I wanted to read the paper anyway, very cool! new information...

Additionally I...

1. moved all annotations to mat3-Mi to mat1-Mi (mat3 is the silent locus)

2. mat1-Mi: 'Transcription factor binding ' annotation:
 * removed the extension has_regulation_target SPBC119.04 (it doens't make sense on this term - and it is redundant with the extension on 'proximal promoter DNA-binding transcription activator activity, RNA polymerase II-specific" anyway
 * changed the term to protein binding
 * made the reciprocal annotation for mat2-Pi

3. mat2-Pi "proximal promoter DNA-binding transcription activator activity, RNA polymerase II-specific" - copied the annotation to mat1-Mi

4. added the GO term 'nuclear chromatin' extension occurs at mei3 for mat1-mi and mat2-pi

5. shk2delta (M cell)- changed 'decreased gene expression' to 'protein absent from cell' (mei3)- I think this is redundant with that there is no karyogamy but I guess technically correct.

6. added 'abolished protein localization to chromatin at promoter' for Midelta & Pidelta affecting Midelta & Pidelta

7. changed the BPs 'positive regulation of meiotic cell cycle ' to 'activation of meiosis'. Also added this term to mat2-Pi

8. added the annotation 'protein localization to nucleus' to mat2-Pi, localizes mat1-Mi

hope that's fine.

**Comment by Antonialock** [comment]
I'll assign to me and do the FAQs

**Comment by Antonialock** [comment]
ah I'm assigned already!

**Comment by ValWood** [comment]
> very cool! 

It is isn't it.

I defer to you on all of the others but:

>
>
> changed the BPs 'positive regulation of meiotic cell cycle ' to 'activation of meiosis'. Also added this term to mat2-Pi

I still think this is regulating entry into the cycle rather than the meiotic nuclear division itself? People often say "regulates meiosis" but  is it really regulating meiosis directly. this is downstream (after DNA replication etc)?
(this is the old semantic issue about the use of the word meiosis)


>
> added the annotation 'protein localization to nucleus' to mat2-Pi, localizes mat1-Mi

Not sure about this one because I got confused again about what-is-what.... but if you are sure then OK

**Comment by Antonialock** [comment]
Ahhh yes this rings a bell. Premeiotic DNA replication is part of meiotic cell cycle. 

I started tidying this but haven't touched it in a while. I think we should have a more specific term than 'positive regulation of meiotic cell cycle ' 
looking back it looks like I settled on cell cycle switching, mitotic to meiotic (and signal transduction involved in cell cycle switching, mitotic to meiotic) (I know you don't like this term)

matpi/matmi upregulates mei3
mei3 inhibits pat1 
inhibition of pat1 activates mei2/sme2
mei2/sme2 somehow activates mei4
mei4 is needed to proceed into meiosis I (I think cells can do premeiotic DNA replication without it but not proceed into meiosis)

cell cycle switching is maybe not the appropriate term for this axis...? 
signal transduction involved in positive regulation of meiosis I?

**Comment by Antonialock** [comment]
FAQ item here: https://github.com/pombase/website/blob/master/frontend/src/docs/faq/how-are-the-mating-type-specific-gene-pages-organized.md

**Comment by Antonialock** [comment]
great!
